ext/opcache/Optimizer/zend_dump.c

```diff
@@ -139,7 +139,7 @@ void zend_dump_var(const zend_op_array *op_array, zend_uchar var_type, int var_n
 		fprintf(stderr, "CV%d($%s)", var_num, op_array->vars[var_num]->val);
 	} else if (var_type == IS_VAR) {
 		fprintf(stderr, "V%d", var_num);
-	} else if (var_type == IS_TMP_VAR) {
+	} else if (var_type == IS_TMP_VAR || !(var_type & (IS_VAR|IS_CV))) {
```

Comment:
I found this a bit confusing. Maybe we can write these two conditions as `var_type & (IS_VAR|IS_TMP_VAR) == IS_VAR` and `var_type & (IS_VAR|IS_TMP_VAR) == IS_TMP_VAR`?